pom.xml

```diff
@@ -132,16 +132,12 @@
                 <version>3.4.2</version>
                 <configuration>
                     <args combine.children="append">
-                        <arg>-deprecation</arg>
+                        <!--arg>-Xlint:deprecation</arg-->
```

Comment:
Maybe create a tracking issue to revert that back in the future (once all warnings are fixed)?

Reply:
The reason why I'm not fixing the warning is that it would break source compatibility with scala 2.11.

Reply:
Agreed, but we probably want to get rid of it "at some point" (once the `android` branch is gone), just want to make sure we don't forget it (we're not looking at the pom very often to be honest)

Reply:
> we're not looking at the pom very often to be honest

Alright. I wouldn't worry about it though, my OCD is already in overdrive with those:

![image](https://user-images.githubusercontent.com/2028222/80082132-3750a300-8554-11ea-9b7e-d00ad2b9ba92.png)

Reply:
Done with https://github.com/ACINQ/eclair/issues/1393.